mmv1/products/backupdr/BackupPlan.yaml

```diff
@@ -0,0 +1,209 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'BackupPlan'
```

Comment:
Could you structure this as two separate PRs, one per resource? Keeping the datasources with their respective resources should be fine. Apologies for the extra work- multi-resource PRs tend to have longer review cycles, and we tend to make more mistakes and/or omissions during those reviews.